Description: avoid opendal-based backends
 Some needed crates are not yet available in Debian.
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: not-needed
Last-Update: 2023-08-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -57,10 +57,8 @@
 number_prefix = "0.4"
 object = "0.32"
 once_cell = "1.19"
-opendal = { version = "0.48.0", optional = true, default-features = false }
 rand = "0.8.4"
 regex = "1.10.3"
-reqsign = { version = "0.16.0", optional = true }
 reqwest = { version = ">= 0.11.27, <= 0.12", features = [
   "json",
   "blocking",
@@ -113,25 +111,9 @@
 [features]
 all = [
   "dist-client",
-  "redis",
-  "s3",
-  "memcached",
-  "gcs",
-  "azure",
-  "gha",
-  "webdav",
-  "oss",
 ]
-azure = ["opendal/services-azblob", "reqsign"]
 default = ["all"]
-gcs = ["opendal/services-gcs", "reqsign", "url", "reqwest/blocking"]
-gha = ["opendal/services-ghac"]
-memcached = ["opendal/services-memcached"]
 native-zlib = []
-oss = ["opendal/services-oss", "reqsign"]
-redis = ["url", "opendal/services-redis"]
-s3 = ["opendal/services-s3", "reqsign", "reqwest"]
-webdav = ["opendal/services-webdav"]
 # Enable features that will build a vendored version of openssl and
 # statically linked with it, instead of linking against the system-wide openssl
 # dynamically or statically.
